student.rb

```diff
@@ -1,15 +1,12 @@
-class Student
-  attr_reader :age, :phase
-  attr_accessor :name
+require_relative 'education'
+
+class Student < Education
 
   def initialize(options = {})
+    # @age = options.fetch(:age, 0)
+    # @name = options.fetch(:name, "")
```

Comment:
Actually remove, don't just comment stuff out.  You have git history if you ever want to look back at what used to be there.